### DISCUSSION TITLE
I can not catch 503(SERVICE_UNAVAILABLE) error when adding a new job to queue

### DISCUSSION THREAD

**Discussion by AppleJin512** [OptimalBits/bull#2600] [Q&A]
Hello, This is Chris!
Nice to meet you.
I'm working on nest.js web app with bull + redis.
My problem is that I'm not able to catch 503 error after calling Queue.add function.
I've deployed a redis server on an online ubuntu server and sometimes get 503 error so I'm now emulating this situation on my local machine. For that I didn't install any redis-docker image on local so that I can get 503 error every time.
But the problem is that I can't catch 503 error when calling Queue.add function. I've tried using try-catch block as well as .catch function but both of them are not working.
Can you plz help me?
I'm quite urgent..

Thanks for reading.

**Comment by manast** [comment]
Most likely not an issue related to BullMQ.

**Reply by manast** [reply]
After re-reading your issue, please check this: https://docs.bullmq.io/patterns/failing-fast-when-redis-is-down

**Reply by AppleJin512** [reply]
Then where can I find the way to resolve this issue? Can you please tell me?

**Reply by manast** [reply]
You got example code in the link I sent you.
